src/test/java/seedu/address/logic/commands/GetEditIngredientCommandTest.java

```diff
@@ -0,0 +1,111 @@
+        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredIngredientList().size() + 1);
+        GetEditIngredientCommand getEditIngredientCommand = new GetEditIngredientCommand(outOfBoundIndex);
+
+        assertCommandFailure(getEditIngredientCommand, model, Messages.MESSAGE_INVALID_INGREDIENT_DISPLAYED_INDEX);
+    }
+
+    //@Test
+    //public void execute_validIndexFilteredList_success() {
+    //    showIngredientAtIndex(model, INDEX_FIRST_INGREDIENT);
+    //
+    //    Ingredient ingredientToDelete =
+    //            model.getFilteredIngredientList().get(INDEX_FIRST_INGREDIENT.getZeroBased());
+    //    DeleteIngredientCommand deleteIngredientCommand = new DeleteIngredientCommand(INDEX_FIRST_INGREDIENT);
+    //
+    //    String expectedMessage = String.format(DeleteIngredientCommand.MESSAGE_DELETE_INGREDIENT_SUCCESS,
+    //            ingredientToDelete);
+    //
+    //    Model expectedModel = new ModelManager(model.getWishfulShrinking(), new UserPrefs());
+    //    expectedModel.deleteIngredient(ingredientToDelete);
+    //    showNoIngredient(expectedModel);
+    //
+    //    assertCommandSuccess(deleteIngredientCommand, model, expectedMessage, expectedModel);
+    //}
+
+    @Test
+    public void execute_invalidIndexFilteredList_throwsCommandException() {
+        showIngredientAtIndex(model, INDEX_FIRST_INGREDIENT);
+
+        Index outOfBoundIndex = INDEX_SECOND_INGREDIENT;
+        // ensures that outOfBoundIndex is still in bounds of fridge's ingredient list
+        assertTrue(outOfBoundIndex.getZeroBased() < model.getWishfulShrinking().getIngredientList().size());
+
+        GetEditIngredientCommand getEditIngredientCommand = new GetEditIngredientCommand(outOfBoundIndex);
+
+        assertCommandFailure(getEditIngredientCommand, model, Messages.MESSAGE_INVALID_INGREDIENT_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void equals() {
+        GetEditIngredientCommand getEditIngredientFirstCommand =
+                new GetEditIngredientCommand(INDEX_FIRST_INGREDIENT);
+        GetEditIngredientCommand getEditIngredientSecondCommand =
+                new GetEditIngredientCommand(INDEX_SECOND_INGREDIENT);
+
+        // same object -> returns true
+        assertTrue(getEditIngredientFirstCommand.equals(getEditIngredientFirstCommand));
+
+        // same values -> returns true
+        GetEditIngredientCommand getEditIngredientFirstCommandCopy =
+                new GetEditIngredientCommand(INDEX_FIRST_INGREDIENT);
+        assertTrue(getEditIngredientFirstCommand.equals(getEditIngredientFirstCommandCopy));
+
+        // different types -> returns false
+        assertFalse(getEditIngredientFirstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(getEditIngredientFirstCommand.equals(null));
+
+        // different x -> returns false
+        assertFalse(getEditIngredientFirstCommand.equals(getEditIngredientSecondCommand));
+    }
+
+    ///**
```

Comment:
Same as previous tests

Reply:
this is used by the test commented out in the same file so since that is commented out this is also commented out